lib/features/home/view/home_view.dart

```diff
@@ -172,7 +204,7 @@ class HomePageViewState extends State<HomePageView> {
                                   onTap: () {
                                     context
                                         .read<HomeInteractionCubit>()
-                                        .onAddPropertyPressed();
+                                        .onTapBottomItems(isUserLoggin);
```

Comment:
same concern: add, booking, msg using the same func onTapBottomItems? I don't know which icon is selected